packages/date-picker/src/FilterDateRangePicker/components/DateRangeValidationMessage/DateRangeValidationMessage.tsx

```diff
@@ -0,0 +1,76 @@
+import React from "react"
+import { FieldMessage } from "@kaizen/draft-form"
+import {
+  DateRangeValidationMessageType,
+  DateRangeValidationStatus,
+  isDateRangeMessageType,
+} from "../../../types"
+import styles from "./DateRangeValidationMessage.module.scss"
+
+export type DateRangeValidationMessageProps = {
+  /**
+   * Updates the styling of the validation FieldMessage
+   */
+  status?: DateRangeValidationStatus
+  /**
+   * A descriptive message for `status` states
+   */
+  validationMessage?: DateRangeValidationMessageType
+  errorMessageId?: string
```

Comment:
```suggestion
  id?: string
```

This is pretty agnostic, and could potentially not be an error :)